RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py

```diff
@@ -58,4 +58,118 @@
   ),
 )
 
+from CondCore.DBCommon.CondDBCommon_cfi import CondDBCommon
+gedelectronGBRESSource = cms.ESSource("PoolDBESSource",
```

Comment:
I'm expecting that this will go away from this PR by the time it's merged.
OK to keep for now while testing.